understanding/low-or-no-background-audio.html

```diff
@@ -143,55 +134,48 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Prerecorded speech is not disrupted by background sound.</dd>
+                  <dt>目標</dt>
+                  <dd>収録済の発話が、背景音によって妨げられないようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Avoid or lessen background sound, or let users turn it off.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>背景音をなくすか小さくする、又は利用者が消せるようにする。</dd>
```

Comment:
avoidは辞書どおり「避ける」でよいのではないかと…？
```suggestion
                  <dd>背景音を避けるもしくは小さくする、又は利用者が消せるようにする。</dd>
```